### ISSUE TITLE
Scheduled sales not working - get_ending_sales() in WC_Product_Data_Store_CPT class file requires braces

### ISSUE THREAD

**Issue by OxfordDigital** [woocommerce/woocommerce#23265]
**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

**To Reproduce**

**Screenshots**

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by OxfordDigital** [comment]
1) Set a sale price and expiry date of today for product variations
2) Run the wc_scheduled_sales() function (or wait until 01:00 next day)
3) Product remains on sale

Occurs with PHP 7.2. Issue is resolved by adding parentheses around time condition in get_ending_sales()

Changed: 
current_time( 'timestamp', true ) - DAY_IN_SECONDS
To: 
(current_time( 'timestamp', true ) - DAY_IN_SECONDS)

Function now returns expired post IDs and correctly expires the sale.

**Comment by mikejolley** [comment]
This is resolved in 3.6 (master) as the offset it no longer there. 3.6 should be out next week.

Thanks
